app/src/main/java/org/mozilla/fenix/components/metrics/Metrics.kt

```diff
@@ -52,6 +52,16 @@ sealed class Event {
             get() = mapOf("source" to source.name)
     }
 
+    data class EnteredUrl(val autoCompleted: Boolean) : Event() {
+        override val extras: Map<String, String>?
+            get() = mapOf("autocomplete" to autoCompleted.toString())
```

Comment:
Honestly I don't love all these booleans-to-strings, because it's less clear what is happening. @mdboom can Glean support non-string, string maps? I thought maps could be printed out as strings without every item being a string.

Reply:
That's a good suggestion, @liuche.  I've filed a [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1538142) to follow up on this.  @boek: I'd suggest you merge this as-is in the PR for now, and we can revisit once the glean team has decided how to move forward with that.  It should be easy enough to find these instances and change them...